.travis.yml

```diff
@@ -9,8 +10,9 @@ notifications:
   webhooks:
     - https://webhooks.gitter.im/e/b5d48907cdc89864b874
 rvm:
-  - 2.5.0
-  - 2.4.1
-  - 2.3.3
-  - 2.2.9
+  - 2.6.1
+  - 2.5.3
```

Comment:
It looks like my code suggestion keep old ruby versions, could you remove 2.6.1 and 2.5.3 ?